lib/internal/test_runner/tap_stream.js

```diff
@@ -15,55 +13,57 @@ const {
   RegExpPrototypeSymbolReplace
 } = require('#internal/per_context/primordials')
 const { inspectWithNoCustomRetry } = require('#internal/errors')
-
+const Readable = require('#internal/streams/readable')
+const { isError } = require('#internal/util')
 const kFrameStartRegExp = /^ {4}at /
 const kLineBreakRegExp = /\n|\r\n/
 const inspectOptions = { colors: false, breakLength: Infinity }
 let testModule // Lazy loaded due to circular dependency.
 
 function lazyLoadTest () {
-  if (!testModule) {
-    testModule = require('./test')
-  }
+  testModule ??= require('#internal/test_runner/test')
 
   return testModule
 }
 
 class TapStream extends Readable {
+  #buffer
+  #canPush
```

Comment:
this syntax won't work on older node versions

Reply:
I does work on Node.js 14.x, I don't see a reason not to use it.

Reply:
TIL!